frontend/src/modules/Shares/components/RevokeShareItemsModal.js

```diff
@@ -10,6 +10,7 @@ import { Defaults } from 'design';
 import { SET_ERROR, useDispatch } from 'globalErrors';
 import { useClient } from 'services';
 import { getShareObject, revokeItemsShareObject } from '../services';
+import { generateShareItemLabel } from '../../../utils/share';
```

Comment:
Same, relative imports should be replaced by global imports

Reply:
Done!